Src/FluentAssertions/Common/AppSettingsConfigurationStore.cs

```diff
@@ -1,4 +1,9 @@
-﻿#if !NETSTANDARD1_3 && !NETSTANDARD1_6
+﻿// Even though .NET Standard 2.0 seems to support the ConfigurationManager class according
+// to the NuGet package at https://www.nuget.org/packages/System.Configuration.ConfigurationManager,
+// it will often throw a PlatformNotSupport exception. See
+// https://docs.microsoft.com/en-us/dotnet/api/system.configuration.configurationmanager?view=netframework-4.8
+
+#if NET47 || NETCOREAPP3_0
```

Comment:
According to https://docs.microsoft.com/en-us/dotnet/api/system.configuration.configurationmanager?view=netframework-4.8, `ConfigurationManager` is not supported on .NET Core 2.0. So even though you may get it to compile, it will most likely through the infamous `PlatformNotSupportedException`.